index.html

```diff
@@ -14,29 +14,386 @@
+              <i class="glyphicon glyphicon-globe"></i>
+              <p>translate</p>
+            </div>
+          </a>
+
+          <a class="no-style" href="mailto:info@ascoderu.ca">
+            <div class="action-thumbnail" data-toggle="tooltip" title="send us an email" data-placement="top">
+              <i class="glyphicon glyphicon-question-sign"></i>
+              <p>ask</p>
+            </div>
+          </a>
+
+          <a class="no-style" href="https://donate.ascoderu.ca/" target='_blank'>
+          <div class="action-thumbnail" data-toggle="tooltip" title="go to our donate page" data-placement="top">
+            <i class="glyphicon glyphicon-usd"></i>
+            <p>donate</p>
+          </div>
+        </a>
+        </div>
+        <a class="no-style medium" href="https://youtu.be/SKtfNZdK_Vw" target='_blank' id="trans-tutorial">
+          <em>see translation tutorial</em>
+        </a>
+
+      </div>
+      <div class="transparent-section-footer">
+        <a href="#news" class="btn btn-lg btn-next" data-toggle="tooltip" title="next page" data-placement="left">
+          <span class="glyphicon glyphicon-menu-down"></span>
+        </a>
+
+      </div>
+    </div>
+
+    <div class="section" data-anchor="news">
+      <div class="container-fluid">
+        <p class="large">What do people say about us?</p>
+        <a class='no-style' href='https://www.howwemadeitinafrica.com/congolese-inventor-solution-get-rural-africa-emailing/59256/' target='_blank'>
+          <div class='box'>
+            <em>Congolese inventor has a solution to get rural Africa emailing</em>,
+            <span>25th Jul 2017 - howwemadeitinafrica.com</span>
+          </div>
+        </a>
+        <a class='no-style' href='http://binspired.ink-live.com/html5/reader/production/default.aspx?pubname=&edid=d2972cbe-118d-40cc-954b-57e3f8f42a72' target='_blank'>
+          <div class='box'>
+            <em>Connect The Dots, Africa business digest p. 102,</em>
+            <span>Aug 2017 - b inspired</span>
+          </div>
+        </a>
+        <a class='no-style' href='https://www.forbes.com/sites/tobyshapshak/2017/06/14/innovation-prize-for-africa-names-its-2017-finalists-for-solving-africas-problems/4/#3dc8c316341f' target='_blank'>
+          <div class='box'>
+            <em>Innovation Prize For Africa Names Its 2017 Finalists For Solving Africa's Problems,</em>
+            <span> 14th Jun 2017 - Forbes</span>
+          </div>
+        </a>
+        <a class='no-style' href='http://www.myjoyonline.com/news/2017/June-16th/aif-announces-the-top-10-nominees-for-innovation-prize-for-africa-2017.php' target='_blank'>
+          <div class='box'>
+            <em>AIF announces top 10 nominees for Innovation Prize for Africa 2017,</em>
+            <span>16th Jun 2017 - Myjoyonline.com</span>
+          </div>
+        </a>
+        <a class='no-style' href=https://face2faceafrica.com/article/nzola-swasisa-lokole' target='_blank'>
```

Comment:
Missing opening quotation mark at the start of the href. `href='https://face2faceafrica.com/...`